pages/avatar/avatar.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar</title>
+</head>
+<link rel="stylesheet" href="/styles/avatar.css">
+<body>
+    <div class="avatar-container">
```

Comment:
The stylesheet link should be in the head and the href path is incorrect. Please check.